pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceProducer.java

```diff
@@ -140,6 +148,9 @@
         @Parameter(names = { "-p", "--max-outstanding-across-partitions" }, description = "Max number of outstanding messages across partitions")
         public int maxPendingMessagesAcrossPartitions = DEFAULT_MAX_PENDING_MESSAGES_ACROSS_PARTITIONS;
 
+        @Parameter(names = { "-np", "--partitions" }, description = "Create partitioned topics with the given number of partitions, set 0 to not try to create the topic")
+        public int createTopicPartitions = 0;
```

Comment:
```suggestion
        public Integer createTopicPartitions =  null;
```

Please use boxed integer to indicate if it is going to create a partitioned topic or not.

Reply:
good idea. done